lib/node_modules/@stdlib/constants/unicode/README.md

```diff
@@ -22,21 +22,33 @@ limitations under the License.
 
 > Unicode constants.
 
-<section class="usage">
-
 ## Usage
 
 ```javascript
 var constants = require( '@stdlib/constants/unicode' );
+
+// Maximum Unicode code point:
+console.log( 'Maximum Unicode code point:', constants.MAX );
+
+// Maximum Basic Multilingual Plane (BMP) code point:
+console.log( 'Maximum BMP code point:', constants.MAX_BMP );
 ```
 
-#### constants
```

Comment:
Why was this change made? Where else in the project do we make the API heading level 2?